docs/cookbook/events_and_gestures.md

```diff
@@ -107,6 +107,19 @@ end
 
 ```
 
+## Custom events
+
+To add a custom event, use `.on` with a custom symbol. Call `.trigger` to trigger your block.
+
+```ruby
+
+# this code allows you to add a custom event and trigger it
+rmq.append(UIView, :my_view).on(:custom_event) do |sender|
+    puts "custom_event has been triggered"
+end
+rmq(:my_view).trigger(:custom_event)
```

Comment:
@squidpunch yep, I started by the code you added, then I look at the rest of the file and saw all rmq related stuff, this is why I wrote the code with rmq, but I close this PR and add a new one with this code